markers/polarion.py

```diff
@@ -13,3 +14,13 @@ def extract_polarion_ids(item):
         return None
 
     return map(str, polarion.args)
+
+
+@pytest.fixture(autouse=True)
+def _update_polarion_in_junit(request, record_xml_property):
+    """Adds the supplied test case id to the xunit file as a property"""
+    from functools import partial
+    record_id_to_xml = partial(record_xml_property, ("test_id",))
```

Comment:
__OPTIONAL__: Please move this definition to the top of the file, i.e.
```python
from functools import partial
```
to the top and
```python
record_id_to_xml = partial(record_xml_property, ("test_id",))
```
below the imports.

Reply:
done